tests/sundial_e2e.spec.ts

```diff
@@ -208,11 +208,11 @@ describe('Sundial Interacting with Port Reserve that has positive APY', () => {
     expect(liquidityWallet.amount.toString()).equal(
       expectedRemainingLiquidity.toString(),
     );
-    expect(principleWallet.amount.add(fee)).to.bignumber.lt(yieldWallet.amount);
+    expect(principleWallet.amount.add(fee)).to.bignumber.eq(yieldWallet.amount);
+    expect(principleWallet.amount.add(fee)).to.bignumber.lt(amount);
     expect(yieldWallet.amount.sub(principleWallet.amount).toNumber()).gt(
       interestAccrue,
```

Comment:
Probably meaningless to have this check because we already have:
```
 expect(principleWallet.amount.add(fee)).to.bignumber.eq(yieldWallet.amount);
```
So in here it is just saying `fee` is greater than `interestAccrue`.